gneiss/plot/_decompose.py

```diff
@@ -0,0 +1,137 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+def balance_boxplot(balance_name, data, num_color='#FFFFFF',
+                    denom_color='#FFFFFF',
+                    xlabel="", ylabel="", linewidth=1,
+                    ax=None, **kwargs):
+    """ Plots a boxplot for a given balance and the associated metadata.
+
+    Parameters
+    ----------
+    balance_name : str
+        Name of balance to plot.
+    data : pd.DataFrame
+        Merged dataframe of balances and metadata.
+    num_color : str
+        Hex for background colors of values above zero.
+    denom_color : str
+        Hex for background colors of values below zero.
+    xlabel : str
+        x-axis label.
+    ylabel : str
+        y-axis label.
+    linewidth : str
+        Width of the grid lines.
+    ax : matplotlib axes object
+        Axes object to render boxplots in.
+    **kwargs : dict
+        Values to pass in to customize seaborn boxplot.
+
+    Returns
+    -------
+    a : matplotlib axes object
+        Matplotlit axes object with rendered boxplots.
+
+    See Also
+    --------
+    seaborn.boxplot
+    """
+    import seaborn as sns
+    if ax is None:
+        f, ax = plt.subplots()
+
+    # the number 20 is pretty arbitrary - we are just
+    # resizing to make sure that there is separation between the
+    # edges of the plot, and the boxplot
+    pad = (data[balance_name].max() - data[balance_name].min()) / 20
```

Comment:
It actually does - but what really sucks here is when you try to do the background shading.

So you do need to have some padding in order to make sure that there is no white space at the ends.   Of course, I would be open to a more clean solution.